recipes/ed25519-blake2b/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ed25519-blake2b" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Matoking/python-ed25519-blake2b/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c5373c2c6bab329753cec746b7b6280995b3406ae648a009cb7e8227537bf024
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - ed25519_blake2b
+  requires:
+    - pip
+    - python
+  source_files:
+    - test_ed25519_kat.py
+    - kat-ed25519-blake2b.txt
+  commands:
+    - pip check
+    - edsig
```

Comment:
This command line script is installed with the package, but it is broken because it relies on the ed25519 package. If ed25519-blake2b is supposed to be co-installable with ed25519, then this script should be removed so there are not clobberings with the ed25519 package. If they are not to be con-installable, then this recipe needs to be run_constrained on `ed25519 ==999999`.

Reply:
I was just looking into it. This is a fork of `python-ed25519` and it does not update the `bin` dir. It seems that it should be removed or updated.
When you say 'should be removed', you're referring to upstream, correct?
I will look into it.

I did not think to test scripts, this is a good lesson on recipe crafting, hope I'll remember and be more thorough

Reply:
Is it ok to add `ed25519` recipe to this PR?

Reply:
You can remove the edsig file from the install list with a small patch to the setup.py. I believe that /bin/edsig exists in both ed25519-blake2b and ed25519, so they would clobber if you install both.

Reply:
You could also add a small patch which fixes the imports in /bin/edsig. I'm not sure which is more appropriate.

Reply:
Right, I am testing a patch that changes the import and the name

Reply:
Looks like it is viable. I went for the patching solution that maintain the script from upstream:
   -  Renamed edsig to edsig_blake2b
   - Patched imports in edsig_blake2b
   - Updated setup.py
   - Updated the README.md